### PR TITLE
cleanup prints

### PR DESCRIPTION
**Pull request by alik-git**
use logging instead of prints

### CODE REVIEW

File: examples/test_extract.py [hunk 1]
```diff
@@ -1,35 +1,33 @@
...
             extracted_krec.save(str(output_path))
-            print(f"Saved to: {output_path}")
+            logging.info(f"Saved to: {output_path}")
 
     except Exception as e:
-        print(f"Error: {e}")
-        return 1
-
-    return 0
+        logging.error(f"Error: {e}")
```
Comment (codekansas) [inline examples/test_extract.py]: oh, i thought i disabled f-strings in logging in pylint... this is not ideal, see here: https://pylint.readthedocs.io/en/stable/user_guide/messages/warning/logging-fstring-interpolation.html

File: examples/test_extract.py [hunk 2]
```diff
@@ -38,5 +36,7 @@ def main(args):
     parser.add_argument("-o", "--output-file", type=str, help="Output KRec file path (optional)")
     parser.add_argument("-v", "--verbose", action="store_true", help="Enable verbose output")
 
+    logging.basicConfig(level=logging.INFO, format="%(asctime)s - %(levelname)s - %(message)s")
```
Comment (codekansas) [inline examples/test_extract.py]: actually, you can add `colorlogging` as a dependency, it is a small package i wrote to make it easy to configure nice logging: https://pypi.org/project/colorlogging/

File: setup.py [hunk 3]
```diff
@@ -60,6 +60,7 @@ def run(self) -> None:
         ),
     ],
     setup_requires=["setuptools-rust"],
+    install_requires=["colorlogging"],
```
Comment (codekansas) [inline setup.py]: put into `requirements.txt` not here

**Comment by alik-git** [comment]
I've merged this branch into [combine_mkv_overwrite_by_default](https://github.com/kscalelabs/krec/tree/combine_mkv_overwrite_by_default) so we can ignore this branch going forward.

**Pull request closed without merging**